### ISSUE TITLE
Expand "index page" and incorporate https://usace.github.io/groundwork/#/docs for consistency.

### ISSUE THREAD

**Issue by MikeNeilson** [USACE/cwms-data-api#572]
This was brought up during another meeting, as we expand the hosted documentation and perhaps some default behavior this will be useful to maintain the USACE look-and-feel consistency.

**Comment by adamkorynta** [comment]
Should also consider the regex explainer page: https://cwms-data.usace.army.mil/cwms-data/regexp.html

**Comment by krowvin** [comment]
@adamkorynta 

I did have a mockup of the new site in react+groundwork and it had a regex page. 
https://github.com/usace-wm-swd/cwms-data-api/blob/data-query-tool/smart-inputs/cda-gui/src/pages/regexp/index.jsx

Also working on a `data-query` page of sorts. 

But we still have an issue with figuring out the base path for this. Since it is not always `cwms-data`.


@MikeNeilson Should I go ahead and bring this branch over to this repo so it's not so hidden? I kept adding features to this but should probably call it somewhere. (Data query tool)

**Comment by adamkorynta** [comment]
@krowvin - great! It would be good to combine given @rma-rripken is adding more pages. See: https://github.com/USACE/cwms-data-api/pull/1169/files#diff-3403f89d85a1e3c41ccf814f67395f78076323243dae3bd98cbd78969756fea7R1

**Comment by MikeNeilson** [comment]
@krowvin yes, and honestly for now we'll just assume cwms-data, that is the future and so how much do we really want to fight it?

@adamkorynta those docs *probably* should go into the new sphinx/read-the-doc based documentation we've started. I'm just not going to worry about that right now as we set that up right in the middle of Ryan creating those.

**Comment by adamkorynta** [comment]
The work has been completed with the groundwork single page app merged into main.
